libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt

```diff
@@ -32,6 +31,13 @@ import com.apollographql.apollo3.compiler.codegen.java.JavaCodeGen
 import com.apollographql.apollo3.compiler.codegen.kotlin.KotlinCodeGen
 import com.apollographql.apollo3.compiler.hooks.ApolloCompilerJavaHooks
 import com.apollographql.apollo3.compiler.hooks.ApolloCompilerKotlinHooks
+import com.apollographql.apollo3.compiler.internal.addRequiredFields
+import com.apollographql.apollo3.compiler.internal.checkApolloInlineFragmentsHaveTypeCondition
+import com.apollographql.apollo3.compiler.internal.checkApolloReservedEnumValueNames
+import com.apollographql.apollo3.compiler.internal.checkApolloTargetNameClashes
+import com.apollographql.apollo3.compiler.internal.checkCapitalizedFields
+import com.apollographql.apollo3.compiler.internal.checkConditionalFragments
+import com.apollographql.apollo3.compiler.internal.checkKeyFields
```

Comment:
Moved to an internal package for the Java callers if any that don't read the Kotlin `internal` modifier.